bundles/org.openhab.ui/web/src/components/item/item-form.vue

```diff
@@ -369,6 +370,17 @@ export default {
         inputElement.dispatchEvent(new Event('input'))
       }
       this.item.label = event.target.value
+    },
+    getCompatibleGroupTypes () {
+      if (!this.itemType) return
+      let compatibleGroupTypes = []
+      compatibleGroupTypes.push(this.item.type)
+      if (this.itemType === 'Number') {
```

Comment:
Why do you use equals `Number` here and startsWith in the group-member?
I would expect to also allow Switch for UoM Items.